tests/specs/mcd/verification.k

```diff
@@ -109,4 +119,53 @@ module VERIFICATION
 
     rule ( BA1:ByteArray ++ BA2:ByteArray ) ++ BA3:ByteArray => BA1 ++ ( BA2 ++ BA3 )
 
+    // ### vat-addui-fail-rough
+
+    // Proof:
+    // case sgn(X) ==Int  1
+    //      X s<Word 0
+    //   => X <Word 0           // sgn(X) ==Int 1 andBool sgn(0) ==Int 1
+    //   => bool2Word(false)    // #rangeUInt(256, X)
+    // case sgn(X) ==Int -1
+    //      X s<Word 0
+    //   => bool2Word(true)     // sgn(X) ==Int -1 andBool sgn(0) ==Int 1
+    rule X s<Word 0 => bool2Word(sgn(X) ==Int -1) requires #rangeUInt(256, X)
```

Comment:
A z3 proof: https://rise4fun.com/Z3/plzQl

(I wish we have a good way to link an external mechanized proof, if any, to each lemma.)

Reply:
I put the Z3 proof in the PR, thanks!